### DISCUSSION TITLE
How to install BepInEx for mods

### DISCUSSION THREAD

**Discussion by pyong-1459** [Nimdy/Dedicated_Valheim_Server_Script#169] [Q&A] [answered]
Hello
I want to know about which directory put the BepInEx file. 
Low data rate cause lag in server, so I want to find where the game folder is.
I cannot find where the game is.
I just find valheim server folder, not the real valheim game folder.
I installed with this script you made.
Where can I find the valheim game installation location?

**Comment by Nimdy** [comment, marked as answer]
I am not a modder. I do not know.

But if you installed the V+ for modding, then the folder is /home/steam/valheimserver/BepInEx/

I can not tell you how to make mods to work for Valheim.  Once I am done with all the other items, I will learn it myself and post information.

If you figure it out, please share this information
